### DISCUSSION TITLE
Invalid client and instant close of crunchy-cli.exe

### DISCUSSION THREAD

**Discussion by NumbPat** [crunchy-labs/crunchy-cli#376] [Q&A]
I'm totaly new to this and tried my best to make it work.
When i open cmd or powershell on my windows 11 pc and try to login into cr, i always getting this error message:

:: √ Logging in
:: invalid_client (auth.obtain_access_token.client_inactive) -  (https://www.crunchyroll.com/auth/v1/token)

I've downloaded the latest release " crunchy-cli-v3.4.1-windows-x86_64.exe " and put it onto my c-drive. I don't know if i did something wrong, but i would be glad, if someone could explaining me what i could do :/

**Comment by bytedream** [comment]
This issue got fixed in [v3.4.2](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.4.2). Download this version to get the fix

**Reply by NumbPat** [reply]
I have to correct myself, that i used 3.4.1. 
I've used 3.4.2 and still getting this error. The one i mentioned.

**Reply by NumbPat** [reply]
Do you know why I still get it? Run powershell normal or in admin, but nothing works. Yes, i do have an account and i followed your tutorial as far as i was able to understand.
